lib/server.js

```diff
@@ -0,0 +1,205 @@
+import http from 'node:http'
+import net from 'node:net'
+import {EventEmitter} from 'node:events'
+import crypto from 'node:crypto'
+import url from 'node:url'
+
+import {Headers} from 'undici'
+
+import {SafeHttpClient, HttpError} from './safe-http-client.js'
+import {
+  DEFAULT_HEADERS,
+  REQUEST_HEADERS,
+  RESPONSE_HEADERS
+} from './constants.js'
```

Comment:
```suggestion
import crypto from 'node:crypto'
import {EventEmitter} from 'node:events'
import http from 'node:http'
import net from 'node:net'
import url from 'node:url'
import {Headers} from 'undici'
import {
  DEFAULT_HEADERS,
  REQUEST_HEADERS,
  RESPONSE_HEADERS
} from './constants.js'
import {HttpError, SafeHttpClient} from './safe-http-client.js'
```

Sorting, no blank lines